public/locales/az/translation.json

```diff
@@ -526,6 +530,7 @@
         "collapse": "Yan paneli yığcamlaşdırın",
         "uncollapse": "Kenar çubuğunu yığışdırın"
     },
+    "Sideload": "Other",
```

Comment:
This as a key might be weird for translators, like... why is this different than the other `Others` added in this file?

Reply:
fixed